CHANGELOG.md

```diff
@@ -37,6 +39,7 @@ _ `_optional` subpackage for managing optional dependencies
 
 ### Fixed
 - `sequential` flag of `SequentialGreedyRecommender` is now set to `True`
+- `best_f` gets now correctly scaled when using non-GP surrogates
```

Comment:
@Scienfitz: this is what we have discussed after @AVHopp left the meeting. Can you please double-check and confirm that it was indeed a bug that is now fixed?

Reply:
```suggestion
- `best_f` now gets correctly scaled when using non-GP surrogates
```